css/style.css

```diff
@@ -0,0 +1,152 @@
+* {
+	box-sizing: border-box;
+	font-family: "Times New Roman", Times, serif;
+}
+
+h1 {
+	margin-bottom: 40px;
+	text-align: center;
+	font-weight: bold;
+	font-family: 'Alice', serif;
+}
+
+section {
+	position: relative;
+	font-weight: bold;
+	}
+
+#boxchicken {
+	background-color: #778899;
+	font-family: 'Alice', serif;
+	position: absolute;
+	border-style: solid;
+	border-width: 1px;
+	font-size: 120%;
+	padding: 5px 30px 5px 30px;
+	top: 0;
+	right: 0;
+	}	
+
+#boxbeef {
+	background-color: #FFB6C1;
+	font-family: 'Alice', serif;
+	font-size: 120%;
+	border-color: 	#000000;
+	border-style: solid;
+	border-width: 1px;
+	padding: 5px 30px 5px 30px;
+	position: absolute;
+	top: 0;
+	right: 0;
+	}	
+
+#boxsushi {
+	background-color: #20B2AA;
+	font-family: 'Alice', serif;
+	font-size: 120%;
+	border-color: 	#000000;
+	border-style: solid;
+	border-width: 1px;
+	padding: 5px 30px 5px 30px;
+	position: absolute;
+	top: 0;
+	right: 0;
+	}
+
+#sectionchicken {
```

Comment:
Same as with the boxes, repeating rules can be moved to the "section" rules.

Reply:
Absolutely true, change done and mental note made!